ktor-server/ktor-server-plugins/ktor-server-partial-content/jvmAndNix/src/io/ktor/server/plugins/partialcontent/PartialContent.kt

```diff
@@ -67,9 +42,14 @@ public val PartialContent: RouteScopedPlugin<PartialContentConfig> = createRoute
     ::PartialContentConfig
 ) {
     onCall { call ->
-        if (call.request.ranges() == null) return@onCall
+        if (call.request.ranges() == null) {
+            LOGGER.trace("Skip partial logging for ${call.request.uri}: no ranges specified")
```

Comment:
`skip partial logging` - is it correct text? (same below)

Reply:
thanks, fixed!